sys-kernel/ugrd/Manifest

```diff
@@ -0,0 +1 @@
+DIST ugrd-1.13.0.tar.gz 63781 BLAKE2B 3f2d3095c6303b16947121071dabdbef81baaea5a7e208d53250e57edf136618497710ce3cca2906b87fbdc529b8a9ef119659365bba8c795594774cf8d096f6 SHA512 a745685668e981accfd32d1c47c6cf506ae63304ef5b3fdad200731db75b76fe56a4339876465bb1857c2fed35c273f95bbce390224fa0d01aebd67adbc31aa6
```

Comment:
Entry for `1.11.0` is missing. Lets just remove that version and keep only the latest. This is the second thing the CI is complaining about: `MissingManifest`

Reply:
My bad, I meant to remove this while updating to the latest version.